added OracleLinux role variables for support

codydiehl opened the pull request:
By default ansible will use CentOS role variables if ran against an
Oracle Linux instance. This will fail the install as the package names
do not match. Instead Oracle Linux packages are equivalent to its Red
Hat counterpart.

rjeffman commented on roles/ipareplica/vars/OracleLinux.yml:
```diff
@@ -0,0 +1,5 @@
+# vars/RedHat-7.yml
+ipareplica_packages: [ "ipa-server", "libselinux-python" ]
+ipareplica_packages_dns: [ "ipa-server-dns" ]
+ipareplica_packages_adtrust: [ "ipa-server-trust-ad" ]
```
Is the package for the replica different than the package for the server?

rjeffman commented on roles/ipaserver/vars/OracleLinux.yml:
```diff
@@ -0,0 +1,5 @@
+# vars/default.yml
+ipaserver_packages: [ "ipa-server", "libselinux-python", "python-six"]
+ipaserver_packages_dns: [ "ipa-server-dns" ]
+ipaserver_packages_adtrust: [ "freeipa-server-trust-ad" ]
```
Is the package for the replica different than the package for the server?

codydiehl replied:
Thank you for caching that. 
My coworker had informed me that he found it necessary during one of his installs, however I have spent the last day and a half testing this 20+ times in a row with various configurations using OracleLinux 7.7 via the official Vagrant box and the ipaserver, ipareplica and ipaclient roles. I am not able to recreate the requirement for python-six, and I have successfully installed all three (server, replica, and client) after dropping that package.

I have removed that package within a new commit I have pushed, as well as cleaned up the comments and also set the vars to target OracleLinux-7 instead of just OracleLinux. 

Please see my updated changes and let me know if there is any concerns. 
Thank you!

rjeffman replied:
Running Oracle Linux 7.7 here (stock, minimal install, adding only the developer_EPEL repo to have ansible available), the name of the package it gives is `ipa-server-trust-ad` and not `freira-server-trust-ad`.

In which version of Oracle Linux is `freipa-server-trust-ad` needed?

codydiehl replied:
I am unable to confirm that, and it may have been overlooked, I have removed it and set it to ipa-server-trust-ad instead after confirming this as well.

codydiehl commented:
This has been updated to reflect the proper and validated changes. 
Can you look it over and verify if there is anything further please? 

Thanks

rjeffman commented:
Would it be possible to merge all the commits for this change into one single commit?

codydiehl commented:
> Would it be possible to merge all the commits for this change into one single commit?

I have rebased and squashed, please look it over when you are able. 
Thanks!

t-woerner commented:
Do you plan to add also files for Oracle Linux 8? Most likely all files for Oracle Linux are simply copies from the files for Red Hat Enterprise Linux.

t-woerner commented:
Please have a look at https://github.com/freeipa/ansible-freeipa/pull/300
This PR is providing the vars files for 7 and 8.

t-woerner commented:
The PR #300 is superseding this PR.
